### DISCUSSION TITLE
MapImage District_of_Rostock

### DISCUSSION THREAD

**Discussion by Desrow** [KillBait/AutoDrive_Course_Editor#40] [General]
Hey there, would you mind creating a Mapimage of this Map or maybe give a hint on how to create one, that would be much appreciated.
[Overview+Heightmap_District_Rostock.zip](https://github.com/KillBait/AutoDrive_Course_Editor/files/8685389/Overview%2BHeightmap_District_Rostock.zip)

**Comment by KillBait** [comment]
Hi... That is done and uploaded for you

A mini guide to setting these up is on my list to do, i'm concentrating on version 1.0 at the moment
so i'll get to it eventually :)

**Reply by Desrow** [reply]
Alright thanks alot that is much appreciated! Awesome work so far i have to say, really like the Tool. If one could only have more resolution of the Map Image it would be perfect. 
Looking forward to 1.0! :) keep it up
